lib/chingu/game_states/pause.rb

```diff
@@ -21,37 +21,37 @@
 
 module Chingu
   module GameStates
-  
+
     #
     # Premade game state for chingu - A simple pause state.
-    # Pause whenever with: 
+    # Pause whenever with:
     #   push_game_state(Chingu::GameStates::Pause)
     #
     # requires the global $window set to the instance of Gosu::Window (automaticly handled if you use Chingu::Window)
     #
     class Pause < Chingu::GameState
-      
+
       def initialize(options = {})
         super
         @white = Gosu::Color.new(255,255,255,255)
         @color = Gosu::Color.new(200,0,0,0)
         @font = Gosu::Font[35]
         @text = "PAUSED - press ESC to return to game."
       end
-    
+
       def button_up(id)
         pop_game_state(:setup => false) if id == Gosu::KbEscape   # Return the previous game state, dont call setup()
       end
-      
+
       def draw
         previous_game_state.draw    # Draw prev game state onto screen (in this case our level)
         $window.draw_quad(  0,0,@color,
                             $window.width,0,@color,
                             $window.width,$window.height,@color,
                             0,$window.height,@color, Chingu::DEBUG_ZORDER)
```

Comment:
Curious how this call has not been reformatted, but it should be replaced by a call to draw_rect anyway.

Reply:
```suggestion
        Gosu..draw_rect 0, 0, $window.width, $window.height, @color, Chingu::DEBUG_ZORDER
```
...but again, better to do this in follow-up